src/schema/v2/conversation/conversations.ts

```diff
@@ -18,22 +48,65 @@ const Conversations: GraphQLFieldConfig<void, ResolverContext> = {
     },
   }).connectionType,
   description: "Conversations, usually between a user and partner.",
-  args: pageable(),
-  resolve: (_root, options, { conversationsLoader, userID }) => {
-    if (!conversationsLoader) return null
-    const { page, size, offset } = convertConnectionArgsToGravityArgs(options)
+  args: pageable({
+    dismissed: {
+      type: GraphQLBoolean,
+    },
+    hasMessage: {
+      type: GraphQLBoolean,
+    },
+    hasReply: {
+      type: GraphQLBoolean,
+    },
+    partnerId: {
+      type: GraphQLString,
+    },
+    type: {
+      type: ConversationsTypeEnum,
+    },
+  }),
+  resolve: (_root, args, { conversationsLoader, userID }) => {
+    if (!conversationsLoader) {
+      return null
+    }
+
+    const { page, size, offset } = convertConnectionArgsToGravityArgs(args)
     const expand = ["total_unread_count"]
+
+    let params
+    if (args.type === "Partner") {
+      if (!args.partnerId) {
+        throw new Error("Argument `partnerId` is required.")
+      }
+
+      params = {
+        deleted: false,
+        intercepted: false,
+        to_id: args.partnerId,
+        to_type: "Partner",
+
+        // TODO
+        dismissed: args.dismissed,
+        has_message: args.hasMessage,
+        has_reply: args.hasReply,
+      }
+      // User
+    } else {
+      params = {
+        expand,
+        from_id: userID,
```

Comment:
at some point this might get extended even further to support an admin use case but this is nice for now!